bofire/data_models/surrogates/surrogate.py

```diff
@@ -28,3 +29,17 @@ def validate_outputs(cls, v, values):
         if len(v) == 0:
             raise ValueError("At least one output feature has to be provided.")
         return v
+
```

Comment:
We need the actual validator that uses the is_output_implemented method, have a look here: https://github.com/experimental-design/bofire/blob/73c1db7876d72e15694610e6b068126da96b98d5/bofire/data_models/strategies/strategy.py#L38 and then we need to test if for at least one surrogate using the `add_invalid` syntax in `specs/surrogates.py`